### ISSUE TITLE
Edit and Delete

### ISSUE THREAD

**Issue by michael132** [twtrubiks/DRF-dataTable-Example-server-side#2]
Hello,

this is more of a question not an issue. I don't see you implement these two methods(Edit and Delete) in views nor add them in urls.py

How do they work?

**Comment by twtrubiks** [comment]
@michael132 Edit and Delete and PUT , In the DRF has been implemented, the  get method needs to be additionally written because of the need to match DataTables .

Edit and Delete and PUT , Please refer to [https://github.com/twtrubiks/django-rest-framework-tutorial](https://github.com/twtrubiks/django-rest-framework-tutorial)

**Comment by Adity14a** [comment]
@twtrubiks Hi, I have implemented your example with mysql, everything works fine but searching sorting and pagination is not working please tell me what could be the reason for this ? I have commented on your youtube channel you gave one github link but not able to understand that. Can you explain me in plain english ?

**Comment by twtrubiks** [comment]
@Adity14a I fixed it, mainly because of the relationship between django>2 and djangorestframework, the new settings are slightly different.
https://github.com/twtrubiks/DRF-dataTable-Example-server-side/tree/django_2_and_mysql_5.7

**Comment by zam20** [comment]
hello @twtrubiks  still don't see delete method in views nor in urls
